modules/gitops-release-notes-1-7-0.adoc

```diff
@@ -0,0 +1,57 @@
+
+* With this update, you can set custom `nodeSelectors` in the Operator workloads by editing their `GitopsService` custom resource. link:https://issues.redhat.com/browse/GITOPS-2164[GITOPS-2164]
+
+
+* With this update, you can choose the RBAC policy matcher mode. There are currently two options `glob` and `regex`. `Glob` is the default.link:https://issues.redhat.com/browse/GITOPS-1975[GITOPS-1975]
+
+* With this update, you can customize resource behavior through the use of additional subkeys. These subkeys are `resourceHealthChecks`, `resourceIgnoreDifferences` and `resourceActions`. Each of the subkeys maps directly to their own field in the `argocd-cm` so `resourceHealthChecks` will map to `resource.customizations.health`, `resourceIgnoreDifferences` to `resource.customizations.ignoreDifferences`, and `resourceActions` to `resource.customizations.actions`.
+
+Eventually, the old method of customizing resource behavior through `resourceCustomization` (without subkeys) will be deprecated.
+
+Keys for `resourceHealthChecks`, `resourceIgnoreDifferences`, and `resourceActions` are in the form (respectively): `resource.customizations.health.<group_kind>`, `resource.customizations.ignoreDifferences.<group_kind>`, and `resource.customizations.actions.<group_kind>`. link:https://issues.redhat.com/browse/GITOPS-1561[GITOPS-1561]
+
+* With this update, to use the *Environments* feature on the *Developer* tab you will be forced to upgrade if you are on {gitops-title} lower than 1.7 and {product-title} higher than or on 4.15. link:https://issues.redhat.com/browse/GITOPS-2415[GITOPS-2415] 
+
+* With this update, applications can be created in any namespace in the same cluster and still managed by the same control-plane’s ArgoCD instance by adding a new label `argocd.argoproj.io/managed-by-cluster-argocd` to the namespace added in `spec.sourceNamespaces` of the Argo CD custom resource. link:https://issues.redhat.com/browse/GITOPS-2341[GITOPS-2341]
+
+:FeatureName: Argo CD Applications controller
+include::snippets/technology-preview.adoc[]
+
+
+[id="fixed-issues-1-7-0_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* From {product-title} 4.12, the console installation will be optional. This fix updates {gitops-title} Operator to prevent problems with the operator if the console is not installed.  link:https://issues.redhat.com/browse/GITOPS-2292[GITOPS-2292]
+
+* Before this update, {gitops-title} releases were affected by an issue with dex pods failing with `CreateContainerConfigError` when a `SCC` of `anyuid` is assigned to the dex `serviceAccount`. This update fixes this issue by assigning a default user id to the dex container. link:https://issues.redhat.com/browse/GITOPS-2235[GITOPS-2235]
```

Comment:
Suggested version:

* Before this update, {gitops-title} releases were affected by an issue of dex pods failing with `CreateContainerConfigError` error when the `anyuid` SCC was assigned to the dex service account`. This update fixes the issue by assigning a default user id to the dex container.